### ISSUE TITLE
error when config file has .cjs extension in jit mode in canary version

### ISSUE THREAD

**Issue by ksmsk** [tailwindlabs/tailwindcss#4504]
### What version of Tailwind CSS are you using?

2.2.0-canary.10

### What build tool (or framework if it abstracts the build tool) are you using?

svelte-kit / vite 2

### What version of Node.js are you using?

v14.17.0

### What browser are you using?

N/A

### What operating system are you using?

linux

### Reproduction repository

https://github.com/ksmsk/tailwind-jit-canary-bug/

### Describe your issue

In `jit` mode there is an error when you run dev server. It looks like it doesn't recognize config file with `.cjs` extension. However it doesn't prevent server running and tailwindcss jit mode still generates classes,  ie: `bg-yellow-700/25`

```
info - Tailwind CSS is watching for changes...
info - https://tailwindcss.com/docs/just-in-time-mode#watch-mode-and-one-off-builds
Error: ENOENT: no such file or directory, access '/$HOME/tailwind-jit-canary-bug/tailwind.config.js'
    at Object.accessSync (fs.js:229:3)
    at resolveConfigPath (/$HOME/tailwind-jit-canary-bug/node_modules/tailwindcss/lib/util/resolveConfigPath.js:52:19)
    at getConfigPath (/$HOME/tailwind-jit-canary-bug/node_modules/tailwindcss/lib/jit/lib/setupWatchingContext.js:103:61)
    at /$HOME/tailwind-jit-canary-bug/node_modules/tailwindcss/lib/jit/lib/setupWatchingContext.js:338:22
    at /$HOME/tailwind-jit-canary-bug/node_modules/tailwindcss/lib/jit/index.js:36:254
    at LazyResult.runOnRoot (/$HOME/tailwind-jit-canary-bug/node_modules/postcss/lib/lazy-result.js:333:16)
    at LazyResult.runAsync (/$HOME/tailwind-jit-canary-bug/node_modules/postcss/lib/lazy-result.js:385:26)
    at LazyResult.async (/$HOME/tailwind-jit-canary-bug/node_modules/postcss/lib/lazy-result.js:215:30)
    at LazyResult.then (/$HOME/tailwind-jit-canary-bug/node_modules/postcss/lib/lazy-result.js:200:17) {
  errno: -2,
  syscall: 'access',
  code: 'ENOENT',
  path: '/$HOME/tailwind-jit-canary-bug/tailwind.config.js'
}
```

**Comment by adamwathan** [comment]
Hey thanks! Looks like this is just a `console.log` statement that got left behind during some debugging, the message will be gone in the next release. As you mentioned everything _is_ working properly, this is just some debug crap hanging around 👍🏻

**Comment by adamwathan** [comment]
Published v2.2.0-canary.11 which should fix this for you 👍🏻
